index.html

```diff
@@ -15,5 +15,88 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <header class="main-nav">
+        <div class="logo">
+            <img src="img/karma-logo.svg" alt="Karma Logo">
+        </div>
+
+        <div class="mobile-menu">
+            <img src="img/menu-hamburger.svg" alt="mobile menu">
+        </div>
+
+        <nav class="main-menu">
+            <ul>
+                <li><a href="#">Meet Karma</a></li>
+                <li><a href="#">How it works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+              </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section class="intro">
+            <div class="intro-img">
+                <div class="intro-text container">
+                    <h1>Introducing Karma</h1>
+                    <p>Bring WiFi with you, everywhere you go</p>
+                    <button class="btn">
+                        learn more
+                    </button>
+                </div>
+            </div>
+        </section>
+
+        <section class="services">
+            <div class="services-text container">
+                <h2>Everyone needs a little Karma</h2>
+            </div>
+
+            <div class="services-box">
+                <div class="service-img">
+                    <figure>
+                        <img src="img/icon-devices.svg" alt="Internet for all devices">
+                        <figcaption>Internet for all devices</figcaption>
+                    </figure>
+                    
+                </div>
+                <div class="service-img">
+                    <figure>
+                        <img src="img/icon-coffee.svg" alt="Coffee cup icon">
+                        <figcaption>Boost your productivity</figcaption>
+                    </figure>
+                    
+                </div>
+                <div class="service-img">
+                    <figure>
+                        <img src="img/icon-refill.svg" alt="charge a device icon">
+                        <figcaption>Pay as you go</figcaption>
+                    </figure>
+                    
+                </div>
+            </div>
+        </section>
+
+        <hr>
+    </main>
+
+    <footer class="footer">
+        
+        <p>Join us</p>
+
+        <div class="social-icons">
+            <img src="img/twitter-icon.svg" alt="Twitter icon">
+            <img src="img/facebook-icon.svg" alt="FaceBook icon">
+            <img src="img/instagram-icon.svg" alt="Instagram Icon">
+        </div>
+
+        <div class="footer-copyrights">
+            <p>&copy; Karma Mobility, Inc.</p>
+        </div>
```

Comment:
```suggestion
        <p class="footer-copyrights">&copy; Karma Mobility, Inc.</p>
```
Same as my other comment